document_page_approval_levels/__openerp__.py

```diff
@@ -30,6 +30,7 @@
         "views/document_page_item_view.xml",
         "views/document_page_view.xml",
         "views/document_page_history_view.xml",
+        "views/document_page_report.xml",
```

Comment:
Hello @LuisAlejandro 
This file(document_page_report.xml) loads actions required by other file, this must be loaded first that document_page_view.xml